src/System.Private.Xml/src/System.Private.Xml.csproj

```diff
@@ -7,7 +7,7 @@
     <AssemblyVersion>4.0.0.0</AssemblyVersion>
     <RootNamespace>System.Xml</RootNamespace>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-    <NoWarn>$(NoWarn),649</NoWarn>
+    <NoWarn>$(NoWarn),649,414</NoWarn>
```

Comment:
What is this warning suppression for?  In general I'd like to see the codebase move away from these project-wide suppressions, as over time we end up hiding warnings that could actually be valuable.  Are these localized enough that pragmas could be used instead at the specific site of the warnings?